ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,244 @@
+
+
+    if cmaps is None:
+        cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
+        cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
+        cmaps = [cmap1, cmap2]
+
+    x, y, z = get_true_volume(b0)
+    coords = (x, y, z)
+
+    labs = [
+        "Sagittal Slice",
+        "Coronal Slice",
+        "Axial Slice",
+    ]
+    var = ["X", "Y", "Z"]
+    # create subplot for first slice
+    # and customize all labels
+    idx = 0
+    if edge:
+        min_val = 0
+        max_val = 1
+    else:
+        min_val, max_val = get_min_max(b0, minthr, maxthr)
+
+    for i, coord in enumerate(coords):
+        for pos in coord:
+            idx += 1
+            ax = foverlay.add_subplot(3, 3, idx)
+            ax.set_title(var[i] + " = " + str(pos))
+            if i == 0:
+                image = ndimage.rotate(b0[pos, :, :], 90)
+                atl = ndimage.rotate(original[pos, :, :], 90)
+            elif i == 1:
+                image = ndimage.rotate(b0[:, pos, :], 90)
+                atl = ndimage.rotate(original[:, pos, :], 90)
+            else:
+                image = ndimage.rotate(b0[:, :, pos], 0)
+                atl = ndimage.rotate(original[:, :, pos], 0)
+
+            if idx % 3 == 1:
+                ax.set_ylabel(labs[i])
+                ax.yaxis.set_ticks([0, image.shape[0] / 2, image.shape[0] - 1])
+                ax.xaxis.set_ticks([0, image.shape[1] / 2, image.shape[1] - 1])
+            if edge:
+                image = edge_map(image).data
+                image[image > 0] = max_val
+                image[image == 0] = min_val
+            # Set the axis invisible
+            plt.xticks([])
+            plt.yticks([])
+
+            # Set the frame invisible
+            ax.spines['top'].set_visible(False)
+            ax.spines['right'].set_visible(False)
+            ax.spines['bottom'].set_visible(False)
+            ax.spines['left'].set_visible(False)
+            ax.imshow(atl, interpolation="none", cmap=cmaps[0], alpha=0.9)
+            ax.imshow(
+                opaque_colorscale(
+                    cmaps[1], image, alpha=0.9, vmin=min_val, vmax=max_val
+                )
+            )
+            if idx ==3:
+                plt.plot(0, 0, "-", c="magenta", label='skull')
+                plt.plot(0, 0, "-", c="green", label='brain')
+                # box = ax.get_position()
+                # ax.set_position([box.x0, box.y0, box.width, box.height*0.8])
+                plt.legend(loc='best', fontsize=15, frameon=False, bbox_to_anchor=(1.5, 1.5))
+
+    # Set title for the whole picture
+    [a, b, c] = ori_shape
+    title = 'QA For skullstrip. Scan Volume:' + str(a) + '*' + str(b) + '*' + str(c)
+    foverlay.suptitle(title, fontsize=24)
+    foverlay.set_size_inches(12.5, 10.5, forward=True)
+    return foverlay
+
+def get_true_volume(nparray):
+    """
+    locates 1/4,1/3,1/2 slices of brain and returns them
+    """
```

Comment:
this needs better docstrings. See any of the other functions in ndmg for examples.
meaning, explain the `Parameters`, and be more descriptive in the description. I don't understand what `locates 1/4,1/3,1/2 slices of brain and returns them` means.

Reply:
fix it, need help.

Reply:
![image](https://user-images.githubusercontent.com/55203069/70833270-59c3fa00-1dc5-11ea-845d-570b99c9ea47.png)